controllers/utils/manifestwork.go

```diff
@@ -119,6 +119,16 @@ func GetManifestWorkForCluster(ctx context.Context, client client.Client, cluste
 	return mw, err
 }
 
+// GetManifestsFromTemplate returns the list of manifests from the manifestwork template
+func GetManifestsFromTemplate(ctx context.Context, client client.Client, name types.NamespacedName) (manifests []mwv1.Manifest, err error) {
```

Comment:
I think a unit test would have been good enough for this function. Kuttl test for this seems like a overkill to me.